src/wallets/hardware/ledger/appPaths.js

```diff
@@ -158,6 +160,11 @@ const appList = [
     network: TT,
     prefixes: ["m/44'/1001'"],
     paths: [thundercore]
+  },
+  {
+    network: PHT,
+    prefixes: ["m/44'/60'"],
+    paths: [lightstreamsNetwork]
```

Comment:
Also,  can you add the path to other wallets, such as `Trezor, keepkey..etc`

Reply:
Addressed at https://github.com/MyEtherWallet/MyEtherWallet/pull/1257/commits/dc31bedd0827b198d9dd64d12f6c9c44401ee541